.hackcambridge/.env.base

```diff
@@ -42,4 +42,3 @@ MIX_PUSHER_APP_KEY="${PUSHER_APP_KEY}"
 MIX_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"
 
 AUTH0_API_AUDIENCE=https://hackcambridge.eu.auth0.com/api/v2/
-
```

Comment:
Whitespace....

I think it will be better to add the newline back to not pollute the history log.